01-Login/README.md

```diff
@@ -1,84 +1,705 @@
-# Scenario #1 - Logging In and Gated Content
+---
```

Comment:
This YAML block can be left out entirely, as it doesn't have any meaning outside of Docs.